README.md

```diff
@@ -9,196 +9,16 @@
-
-Yes!
-
-One of the huge benefits of using TypeScript is the fact that type information can be used to assert expected behaviors.
-
-When the transformation steps outlined above take place, we keep references to the original TypeScript AST and associated parser services, and so ESLint rules authors can access them in their rules.
-
-We already do this in numerous rules within [`@typescript-eslint/eslint-plugin`](./packages/eslint-plugin/), for example, `no-unnecessary-type-assertion` and `no-inferrable-types`.
-
-<br>
-
-## What about Babel and `@babel/eslint-parser`?
-
-Babel does now support parsing (but not type-checking) TypeScript source code. This is as an alternative to using the TypeScript Compiler. It also supports many other syntaxes, via plugins, which are not supported by the TypeScript Compiler. As mentioned above, `typescript-eslint` is powered by the TypeScript Compiler, so we support whatever it does.
-
-The key trade-off can be summarized as `@babel/eslint-parser` supports additional syntax which TypeScript itself does not, but `typescript-eslint` supports creating rules based on type information, which is not available to babel because there is no type-checker.
-
-Because they are separate projects powered by different underlying tooling, they are currently not intended to be used together.
-
-Some of the people involved in `typescript-eslint` are also involved in Babel and `@babel/eslint-parser`, and in this project, we are working hard to align on the AST format for non-standard JavaScript syntax. This is an ongoing effort.
-
-<br>
-
-## How can I help?
-
-I'm so glad you asked!
-
-As you can see at the [top of this repo](#typescript-eslint), these packages are already downloaded millions of times per month, and power high profile projects across our industry.
-
-Nevertheless, this is a 100% community-driven project. From the second you install one of the packages from this monorepo, you are a part of that community.
-
-Please be respectful and mindful of how many hours of unpaid work go into building out all of the functionality we have introduced (in brief detail) above.
-
-We can always do better, but providing the glue between two different tools is always extra difficult because both sides come with their own assumptions and priorities.
-
-See an issue? Report it in as much detail as possible, ideally with a clear and minimal reproduction. Think about what information you would need to start solving the problem yourself and take it from there.
-
-If you have the time and the inclination, you can even take it a step further and submit a PR to improve the project.
-
-There are also financial ways to contribute, please see [Financial Contributors](#Financial-Contributors) for more details.
-
-All positive contributions are welcome here!
+<p align="center">
+  <big>
+  See <strong><a href="https://typescript-eslint.io">typescript-eslint</a></strong> for documentation.
+  </big>
+</p>
```

Comment:
Proposal: now that the docs page covers basic use cases, it'd be better to specify just the GitHub-area info like contributing and versioning.